velox/dwio/dwrf/writer/Writer.h

```diff
@@ -32,12 +32,7 @@ namespace facebook::velox::dwrf {
 
 struct WriterOptions : public dwio::common::WriterOptions {
   std::shared_ptr<const Config> config = std::make_shared<Config>();
-  std::shared_ptr<const Type> schema;
```

Comment:
Why remove these? Thanks!

Reply:
@xiaoxmeng The dwio::common::WriterOptions class already has definitions for these five variables [here](https://github.com/facebookincubator/velox/blob/main/velox/dwio/common/Options.h#L597-L608), so there is no need to redefine them in the subclass. If they were defined here, when casting from dwio::common::WriterOptions to dwrf::WriterOptions, these five variables in the subclass would not be automatically initialized with the definitions from the parent class. Manual conversion would be required [here](https://github.com/facebookincubator/velox/blob/main/velox/dwio/dwrf/writer/Writer.cpp#L848-L851), which is what this code here seems to be doing. It appears to be unnecessary.